src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/Maintainability/ReviewUnusedParameters.Fixer.cs

```diff
@@ -28,19 +31,61 @@ public sealed override FixAllProvider GetFixAllProvider()
             return WellKnownFixAllProviders.BatchFixer;
         }
 
-        public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
+        public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
         {
+            Document document = context.Document;
+            CancellationToken cancellationToken = context.CancellationToken;
+            SemanticModel semanticModel = await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(false);
+            SyntaxNode root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
             foreach (var diagnostic in context.Diagnostics)
             {
+                SyntaxNode diagnosticNode = root.FindNode(diagnostic.Location.SourceSpan);
+                SyntaxNode declarationNode = GetParameterDeclarationNode(diagnosticNode);
+                if (semanticModel.GetDeclaredSymbol(declarationNode, cancellationToken) is not IParameterSymbol parameterSymbol)
+                {
+                    Debug.Fail("Symbol was not IParameterSymbol.");
+                    continue;
+                }
+
+                if (parameterSymbol.ContainingSymbol is not IMethodSymbol methodSymbol)
+                {
+                    Debug.Fail("ContainingSymbol of IParameterSymbol was not IMethodSymbol.");
+                    continue;
+                }
+
+                if (IsSafeMethodToRemoveParameter(methodSymbol))
+                {
+                    // See https://github.com/dotnet/roslyn-analyzers/issues/1466
+                    context.RegisterCodeFix(
+                        new MyCodeAction(
+                            MicrosoftCodeQualityAnalyzersResources.RemoveUnusedParameterMessage,
+                            async ct => await RemoveNodes(document, declarationNode, parameterSymbol, ct).ConfigureAwait(false),
+                            equivalenceKey: MicrosoftCodeQualityAnalyzersResources.RemoveUnusedParameterMessage),
+                        diagnostic);
+                }
+
                 context.RegisterCodeFix(
```

Comment:
When registering multiple codefixes, is it guaranteed they'll appear in the same order in the bulb-light window?